github/resource_github_actions_secret.go

```diff
@@ -16,13 +16,13 @@ func resourceGithubActionsSecret() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceGithubActionsSecretCreateOrUpdate,
 		Read:   resourceGithubActionsSecretRead,
-		Update: resourceGithubActionsSecretCreateOrUpdate,
```

Comment:
Can you explain why removing the update and adding ForceNew solves this issue?

Reply:
The Resource Id is made up of the `repository` and `secret_name` as a `TwoPartID` which should be immutable if im not mistaken.

While the update is removed because all properties have forcenew (terraform will complain about it if its left in).

Reply:
`github/resource_github_actions_environment_secret.go` seems to be similar

Reply:
Ahh got it, thank you!